src/nimble.nim

```diff
@@ -484,10 +484,17 @@ proc downloadPkg(url: string, verRange: VersionRange,
   ## which was downloaded.
   let downloadDir = (getNimbleTempDir() / getDownloadDirName(url, verRange))
   createDir(downloadDir)
-  let modUrl =
+  var modUrl =
     if url.startsWith("git://") and options.config.cloneUsingHttps:
       "https://" & url[6 .. ^1]
     else: url
+
+  # Fixes issue #204
+  # github + https + trailing url slash causes a
+  # checkout/ls-remote to fail with Repository not found
+  if modUrl.contains("https://github.com") and modUrl.endswith("/"):
```

Comment:
what about ``git://github.com``? Might be better to just check for ``github.com``.